cfme/tests/infrastructure/test_tenant_quota.py

```diff
@@ -107,10 +107,10 @@ def small_vm(provider, small_template_modscope):
 @pytest.fixture()
 def check_hosts(small_vm, provider):
     """Fixture to return host"""
-    if len(provider.hosts) != 1:
+    if provider.hosts != 1:
```

Comment:
Required: this will be always true as provider.hosts is hosts collection object
you need to use provider.hosts.all() this will return you a list all available hosts and len(provider.hosts.all()) will do the trick

Reply:
Thanks